apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.py

```diff
@@ -837,25 +891,43 @@ def test(element_html: str, data: pl.ElementTestData) -> None:
     # TODO: The only wrong answer being tested is the correct answer with the first
     # block mising. We should instead do a random selection of correct and incorrect blocks.
     elif data["test_type"] == "incorrect":
-        answer = filter_multiple_from_array(
-            data["correct_answers"][answer_name], ["inner_html", "indent", "uuid"]
-        )
+        answer = deepcopy(data["correct_answers"][answer_name])
+        distractor_feedback = {
```

Comment:
Looks like the distractor feedback dicts are using different keys when they're constructed in different places?

Reply:
cleaned up